src/main/java/org/opentripplanner/middleware/otp/OtpDispatcher.java

```diff
@@ -1,8 +1,74 @@
 package org.opentripplanner.middleware.otp;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.core.UriBuilder;
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.time.Duration;
+
+import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
+
 /**
- * Defines the available interfaces to OTP
+ * Responsible for constructing requests to an elected OTP server end point using original query parameters provided by
+ * the requester. To provide the response from the OTP server in the form of status code and body so the correct
+ * response can be provided to the requester.
  */
-public interface OtpDispatcher {
-    OtpDispatcherResponse getPlan(String query, String endPoint);
+public class OtpDispatcher {
+    private static final Logger LOG = LoggerFactory.getLogger(OtpDispatcher.class);
+    private static String OTP_SERVER = getConfigPropertyAsText("OTP_SERVER");
+    private static final int OTP_SERVER_REQUEST_TIMEOUT_IN_SECONDS = 10;
+
+    /**
+     * Provides a response from the OTP server target service based on the query parameters provided.
+     */
+    public static OtpDispatcherResponse serviceRequest(String query, String endpoint) {
```

Comment:
I'm not too fond of this method's name (nor the name of `call`). Maybe something a bit more descriptive? Perhaps just the use of `service` as a verb feels a bit far out of the American vernacular :)
- serviceRequest -> sendOtpRequest
- call -> sendOtpRequest

It seems like these two methods do effectively the same thing, so having them overloaded with different args feels right to me.

Reply:
Updated.